summarisation_map_reduce.py

```diff
@@ -0,0 +1,56 @@
+from langchain.chains.mapreduce import MapReduceChain
+from langchain.text_splitter import CharacterTextSplitter
+from langchain.chains import ReduceDocumentsChain, MapReduceDocumentsChain
+
+llm = ChatOpenAI(temperature=0)
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ChatOpenAI` class is not imported. This could lead to a `NameError` when trying to create an instance of `ChatOpenAI`. Please import the necessary class.
```diff
+ from langchain.models import ChatOpenAI
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->